src/renderer/pages/ConnectionView.tsx

```diff
@@ -77,21 +105,77 @@ const ConnectionView = (): JSX.Element => {
     ipcRenderer.send(VIEW_CONNECTION_LIST);
   };
 
+  const exportLogs = () => {
+    const blob = new Blob([JSON.stringify(filteredLogs)], {
+      type: 'application/json',
+    });
+    const link = document.createElement('a');
+    link.href = window.URL.createObjectURL(blob);
+    link.download = 'logs.json';
+    link.click();
+  };
+
+  const formatLog = (
+    msg: ConnectionStatusUpdate,
+    remoteAddr: string
+  ): SimplifiedLog => {
+    const date = new Date().toISOString();
+    const status = msg.lastError ? 'error' : 'info';
+    let message = '';
+
+    switch (msg.status) {
+      case 1:
+        message = msg.peerAddr + ' opening connection to ' + remoteAddr;
+        break;
+      case 2:
+        message =
+          msg.peerAddr +
+          ' authentication with ' +
+          msg.authUrl +
+          ' required for ' +
+          connection.remoteAddr;
+        break;
+      case 3:
+        message = msg.peerAddr + ' connected to ' + remoteAddr;
+        break;
+      case 4:
+        message = msg.peerAddr + ' disconnected from ' + remoteAddr;
```

Comment:
I'm not sure if repetitive `remoteAddr` mention is justified - logs are only visible for a particular connection where you already see the remote address. 

<img width="916" alt="image" src="https://user-images.githubusercontent.com/673131/145107024-eb96067d-9ef6-4782-8868-f847fb9c009c.png">

@travisgroth

Reply:
agreed we could remove them